utilities/rocAL/rocAL_performance_tests_with_depth/rocAL_performance_tests_with_depth.cpp

```diff
@@ -411,7 +411,8 @@ int test(int test_case, const char* path, int rgb, int gpu, int width, int heigh
             for(int j = 0; j < batch_size; j++){
                 image0 = inputImage;
                 for(int k = 0; k < graph_depth; k++){
-                    image0 = rocalSnPNoiseFixed(handle, image0, 0.5, true);
+                    image0 = rocalSnPNoiseFixed(handle, image0, true, 0.2, 0.2, 0.2, 0.5, 0);
```

Comment:
0.2, 0.2, 0.2, 0.5, 0 ? why this is addede?

Reply:
These are some values passed to the arguments newly added which doesn't have default args